kafka-client/src/main/java/dev/responsive/kafka/internal/db/FactSchemaWriter.java

```diff
@@ -28,33 +28,33 @@ public class FactSchemaWriter<K> implements RemoteWriter<K> {
 
   private final CassandraClient client;
   private final RemoteSchema<K> schema;
-  private final String name;
-  private final int partition;
+  private final String tableName;
+  private final int subpartition;
```

Comment:
Sorry for all the renaming here, I know it makes PR reviews more annoying but I kept having to jump around to understand when something was a kafka partition vs a subpartition. Hopefully this helps others follow the code the way it helps me

Reply:
I was thinking of it as a subpartition that just happened to be 1:1 with the kafka partitions, but I guess that's confusing too -- the goal was to distinguish between the kafka partition vs the table partition, regardless of the mapping between them

Idk how I didn't think of `tablePartition` vs `kafkaPartition` though, I like that a lot. Great suggestion!